### ISSUE TITLE
how to use the function "register_scale_and_shift" 

### ISSUE THREAD

**Issue by Yuz-Wang** [tensorflow/kfac#29]
The function "auto_register_layers" will raise error, when I add bach_normalization layer. How to deal with it?

**Comment by james-martens** [comment]
What's the error?

**Comment by james-martens** [comment]
Batch normalization now has better support with auto_register_layers than it did before.
